sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -889,6 +896,123 @@ public void testGroupBySingleColumnDescendingNoTopN() throws Exception
     );
   }
 
+  @Test
+  public void testEarliestAggregators() throws Exception
+  {
+    // Cannot vectorize EARLIEST aggregator.
+    skipVectorize();
```

Comment:
Does this need to be reset at the end of the test? Otherwise all the other tests that run after this will run with vectorize disabled? Or is that being handled by some rule that I'm not seeing in this delta

Reply:
The test framework creates a new instance for each test method, so that takes care of resetting it.